contributors/design-proposals/auth/apparmor.md

```diff
@@ -63,7 +63,7 @@ and is supported on several
 # Alpha Design
 
 This section describes the proposed design for
-[alpha-level](../devel/api_changes.md#alpha-beta-and-stable-versions) support, although
+[alpha-level](https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions) support, although
```

Comment:
Is there a reason this uses the shortener, as opposed to just `/contributors/devel/sig-architecture/api_changes.md`?

Reply:
No reason. I'll fix this.

Reply:
There's a couple more of these.. Can you [review](https://github.com/kubernetes/community/pull/3151/files) and correct in the other places too?

Reply:
I'll review from the root folder again. Thanks.